### PR TITLE
bootstrapping docs

### PR DESCRIPTION
**Pull request by ajwootto**
- add docs for using Node JS to bootstrap client side SDKs

### CODE REVIEW

File: docs/sdk/server-side-sdks/node/node-bootstrapping.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: Bootstrapping / Server-Side Rendering 
+sidebar_label: Bootstrapping / SSR
+sidebar_position: 5
+description: SDK features for bootstrapping a client SDK on the server
+---
+
+# Bootstrapping and Server-Side Rendering
+When using a server rendering framework such as Remix, Nuxt, or SvelteKit, you will likely be rendering content on the server and sending it to the client for hydration. When feature flagging is involved, you need to make sure that rendering on the server uses the same flag values as the client. It is also important to avoid the performance impact of the initial clientside DevCycle configuration fetch that would normally have to occur when the page is first loaded.
```
Comment (cdruxerman) [inline docs/sdk/server-side-sdks/node/node-bootstrapping.md:9]: client-side

File: docs/sdk/server-side-sdks/node/node-bootstrapping.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+To support these use-cases, the Node.js SDK provides functionality for generating clientside configurations on the server, for use during server-side rendering as well as bootstrapping on the client.
```
Comment (cdruxerman) [inline docs/sdk/server-side-sdks/node/node-bootstrapping.md:11]: client-side

**Review by cdruxerman — COMMENTED** [review]
Left a couple small comments, but generally good

**Comment by cdruxerman** [comment]
@ajwootto the bootstrapping card needs an icon on the node sdk landing page
Screenshot 2024-05-03 at 2 50 29 PM

**Comment by ajwootto** [comment]
* **#656** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/devcycle-docs/656?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ajwootto and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ajwootto** [comment]
> @ajwootto the bootstrapping card needs an icon on the node sdk landing page Screenshot 2024-05-03 at 2 50 29 PM

done

**Review by cdruxerman — APPROVED** [review]

**Pull request merged**